### DISCUSSION TITLE
Can it extend on mobile phones

### DISCUSSION THREAD

**Discussion by Harshit-Gupta-2005** [Daksh777/SpotifyNoPremium#100] [Q&A]
I enabled it on my laptop but curious about it having the same effect on mobile app (since the account is same)

also, will using spotify mobile reset it (do I need to reenable it after using spotify mobile?)

**Comment by Daksh777** [comment]
You can use this modded Spotify app for Android: https://github.com/xManager-App/xManager
What do you mean by Spotify being reset?

**Reply by Harshit-Gupta-2005** [reply]
By 'resetting Spotify ' I meant 'will the ads and the banners show up again?'
But that link (ig) would help me
Thank you
Also, can I still connect spotify on my phone to my laptop after using it
(Sorry for late reply)

**Reply by Daksh777** [reply]
Oh no, the modded app wouldn't affect your Spotify installation on desktop and you can still connect your phone to your desktop in Spotify
